src/rust/engine/engine_cffi/src/lib.rs

```diff
@@ -966,6 +966,27 @@ pub extern "C" fn run_local_interactive_process(
           Some(TempDir::new().map_err(|err| format!("Error creating tempdir: {}", err))?)
         };
 
+        let input_files_value = externs::project_ignoring_type(&value, "input_files");
+        let digest: Digest = nodes::lift_digest(&input_files_value)?;
+        if digest != EMPTY_DIGEST {
+          if run_in_workspace {
+            return Err("When running an interactive process in the workspace, use the Workspace type to materialize files.".to_string());
+          } else {
+            let destination = match maybe_tempdir {
+              Some(ref dir) => dir.path().to_path_buf(),
+              None => unreachable!()
+            };
+
+            let write_operation = scheduler.core.store().materialize_directory(
+              destination,
+              digest,
+              session.workunit_store().clone(),
+            );
+
+            let _ = scheduler.core.executor.spawn_on_io_pool(write_operation).wait()?;
```

Comment:
Why the `let _`?

Reply:
I thought this was necessary to avoid a compiler warning about not using the return value from `spawn_on_io_pool`, but I might've been wrong about that.

Reply:
You might not need it: `materialize_directory` does return some stats, but I think that because it's not marked `must_use`, it won't be a warning to discard it.

Reply:
You're right; removed this